eks/aws-node-termination-handler/Settings.dhall

```diff
@@ -8,7 +8,8 @@ let Pod =
       let Image =
             { Type = { repository : Text, tag : Text, pullPolicy : Text }
             , default =
-              { repository = "amazon/aws-node-termination-handler"
+              { repository =
+                  "625240141681.dkr.ecr.eu-west-1.amazonaws.com/amazon-aws-node-termination-handler"
```

Comment:
Remember that this repository is open-source. Private ECR is overridden in our private repository, not in the open-source repository